test/BaseTokenWrapper.t.sol

```diff
@@ -50,7 +56,7 @@ abstract contract BaseTokenWrapperTest is Test {
 
   function testConstructor() public virtual;
 
-  function testSupplyToken() public {
+  function testSupplyToken2() public {
```

Comment:
Why this rename? Can we make it something clearer?

Reply:
my bad, let's revert this change